lib/Transforms/InstCombine/InstructionCombining.cpp

```diff
@@ -3028,7 +3030,7 @@ PreservedAnalyses InstCombinePass::run(Function &F,
   auto *LI = AM->getCachedResult<LoopAnalysis>(F);
 
   // FIXME: The AliasAnalysis is not yet supported in the new pass manager
-  if (!combineInstructionsOverFunction(F, Worklist, nullptr, AC, TLI, DT, LI))
+  if (!combineInstructionsOverFunction(F, Worklist, false /*HLSL Change*/, nullptr, AC, TLI, DT, LI))
```

Comment:
It's a nit and maybe not worth delaying further, but we often put something like /*NoSink*/ on literal arguments

Reply:
Fixed!